release/air_tests/air_benchmarks/workloads/xgboost_benchmark.py

```diff
@@ -132,7 +126,7 @@ def __call__(self, data: pd.DataFrame) -> Dict[str, np.ndarray]:
         # batch size than default 4096
         batch_size=8192,
         compute=ray.data.ActorPoolStrategy(min_size=1, max_size=None),
-        fn_constructor_kwargs={"checkpoint": ckpt},
+        fn_constructor_kwargs={"model": model},
```

Comment:
I think the purpose of this is to showcase the passing of a checkpoint, but at the same time it isn't really necessary for a benchmark.

Reply:
I think this was passing the full model through object store before as well bc of checkpoint serialization craziness

I was also considering changing it to create a generic ckpt and passing it in so that it downloads on each map batches actor separately